### PR TITLE
Add per-project work types override.

### PR DESCRIPTION
**Pull request by Moo64c**
#238 
- [x] Enable per project work types override.
- [x] Create account-default work types.

@bricel 
Halfway done, I'll try to finish this in the evening.

### CODE REVIEW

**Comment by Moo64c** [comment]
selection_053
selection_052
selection_051
selection_050

**Comment by Moo64c** [comment]
@bricel 
Ready for CR.
Added ``entityreference_autofill`` module. Works great. Code looks good, however, have not been updated lately.

File: productivity/modules/custom/productivity_project/productivity_project.info [hunk 1]
```diff
@@ -37,12 +37,14 @@ features[field_base][] = field_status
...
 features[field_base][] = field_team_leads
 features[field_base][] = field_type
+features[field_base][] = field_work_type_display
 features[field_group][] = group_deprecated|node|project|form
 features[field_instance][] = multifield-field_table_rate-field_hours
 features[field_instance][] = multifield-field_table_rate-field_issue_type
 features[field_instance][] = multifield-field_table_rate-field_rate
 features[field_instance][] = multifield-field_table_rate-field_rate_type
 features[field_instance][] = multifield-field_table_rate-field_scope
+features[field_instance][] = multifield-field_table_rate-field_work_type_display
```
Comment (bricel) [inline productivity/modules/custom/productivity_project/productivity_project.info]: Better to use the name issue_type_label

**Comment by Moo64c** [comment]
@bricel Ready.

**Pull request merged**